README.md

```diff
@@ -1,20 +1,23 @@
-# Origami
-
-<p align="center"><img width="45%" height="15%" src="origami.png" /></p>
+<p align="center"><img width="30%" height="15%" src="origami_logo.png" /></p>
 
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
```

Comment:
Why?

Reply:
center tag was not working thats why i used &nbsp.

Reply:
https://github.com/anfederico/Clairvoyant , they  follow same styling.